### ISSUE TITLE
calculate M matrix per time integration

### ISSUE THREAD

**Issue by nkern** [HERA-Team/hera_pspec#73]
in the case that flags are time dependent, we need to re-calculate the fisher matrix `G` (which will change the normalization matrix `M`) at every single time integration. provide a master override flag to calculate `G` only once per blpair to speed things up

**Comment by nkern** [comment]
[7/6/2018] Deferred for now...
